tests/asgi/tests.py

```diff
@@ -2,14 +2,17 @@
 import sys
 import threading
 import time
+from contextlib import (
+    nullcontext,  # Used in test_read_body_comprehensive_cases (ticket #36281)
+)
```

Comment:
Please single-line this and remove the comment. 

```suggestion
from contextlib import nullcontext
```

Reply:
Hi @carltongibson , thank you for the detailed review!

You're absolutely right — the current test doesn't directly assert the thread context of the write() call.

I'll revise the test to patch the write method and explicitly check which thread it's being called from (in-memory vs. on-disk). Once I’ve got that in place, I’ll update the PR accordingly.

Appreciate your guidance as always!